### PR TITLE
Add minimize button on each beneficiaries card and search button for beneficiaries list 

### PR DESCRIPTION
**Pull request by fakhrihawari**

### CODE REVIEW

File: app/scripts/modules/cluster/views/forms/details/target-beneficiaries/target-beneficiaries.html [hunk 1]
```diff
@@ -67,11 +108,30 @@ <h5 class="report-beneficiary-title" ng-if="beneficiary.activity_type_id && bene
...
+								</md-tooltip>
+							</button>
+							<button id="closeDB-{{$index}}"
+								ng-show="detailBeneficiaries[$index]"
+								class="btn-floating "								 
+								ng-click="project.openCloseDetailBeneficiaries($index)">
+								<i class="material-icons">keyboard_arrow_up</i>
+								<md-tooltip md-direction="right">
+									Minimize
+								</md-tooltip>
```
Comment (drfaustusfade) [inline app/scripts/modules/cluster/views/forms/details/target-beneficiaries/target-beneficiaries.html]: what do you think if using evaluate expression like below?
```
<button
	class="btn-floating"								 
	ng-click="project.openCloseDetailBeneficiaries($index)">
	<i class="material-icons">{{ detailBeneficiaries[$index] ? 'keyboard_arrow_up' : 'keyboard_arrow_down' }} </i>
	<md-tooltip md-direction="right">
		{{ detailBeneficiaries[$index] ? 'Minimize' : 'Click to Expand' }}
	</md-tooltip>
</button>
```

File: app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js [hunk 2]
```diff
@@ -684,6 +694,28 @@ angular.module( 'ngm.widget.project.details', [ 'ngm.provider' ])
 					});
 				}, 
 
+				// open-close beneficiary target detail
+				openCloseDetailBeneficiaries:function(index){
+					if (!$scope.detailBeneficiaries[index]){
+						$('#openDB-' + index).hide();
+						$('#closeDB-' + index).show();
```
Comment (drfaustusfade) [inline app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js]: if using ng-show on buttons, can we remove jquery hide/show?
```
$('#openDB-' + index).hide();
$('#closeDB-' + index).show();
```

Comment (fakhrihawari) [reply app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js]: With new commit, i remove the rightnow remove the jquery hide and show

**Comment by fakhrihawari** [comment]
If beneficiary.activity_detail_name exist it make the title too long and will look like this
toolaon

**Comment by pfitzpaddy** [comment]
font 20px might work (same as the beneficiaries number)

**Comment by pfitzpaddy** [comment]
A couple of minor udpates https://github.com/pfitzpaddy/ngm-reportHub/commit/18bbba837cd944e8451e324826457fc90dd06131

**Pull request merged**